.eslintrc.js

```diff
@@ -1,46 +1,299 @@
-// This is a convenience field for what rules we want to use
-const rules = {
-  'semi': ['error', 'always'],
+/**
+ * This file represents the best approximation of our development conventions in this project as of
```

Comment:
Not sure this comment is relevant as this repo has never used tslint

Reply:
Good point, I copy/pasted thinking I would keep the comment "as is" but then ended up adding some to it anyways. I'll remove this bit.